python/test/unit/language/test_tuple.py

```diff
@@ -162,3 +162,16 @@ def mul(x, a):
     ty = Tensor(y, y.shape, y.stride())
     _namedtuple_kernel[(1, )](function, tx, ty, 64, 64)
     assert torch.allclose(y, x[:16, :16] * a)
+
+
+# -------
+
+
+@triton.jit
+def _namedtuple_float_tuple_kernel():
```

Comment:
It seems more like a general test rather than something related with tuple. I would suggest moving it to `test_core` without using tuples.